transaction/orders/query.js

```diff
@@ -26,15 +26,15 @@ class queryOrders extends persistenceClass {
                     reject(error);
                 }
                 let result = JSON.parse(res.body)
-                let list = result.result.value.orders.value.list
+                let list = result.result.list
                 if(list != null) {
                     list.forEach(function (value) {
                         if (value.value.immutables.value.properties.value.propertyList[0].value.id.value.idString === id) {
-                            ordersData.clasificationID = value.value.id.value.classificationID.value.idString
-                            ordersData.makerownableid = value.value.id.value.makerOwnableID.value.idString
-                            ordersData.takerownableid = value.value.id.value.takerOwnableID.value.idString
-                            ordersData.makerID = value.value.id.value.makerID.value.idString
-                            ordersData.hashID = value.value.id.value.hashID.value.idString
+                            ordersData.clasificationID = value.value.key.value.classificationID.value.idString
+                            ordersData.makerownableid = value.value.key.value.makerOwnableID.value.idString
```

Comment:
if we only require this for test.js, can we move it somewhere else?

Reply:
Its being used as a comon query and loop

Reply:
this is the function which queries all the orders, and searches for a specific propertyID, right?
in line 32 you are just searching for the 0th element of property list, the intended propertyID could be at some other index, you might have to loop through entire property list so that the function gives expected result, and when there are more than one orders with same propertyID even then it will give random result
We don't need this function in JS. just normal query is enough.

Reply:
Done